setup.cfg

```diff
@@ -1,2 +1,44 @@
+[metadata]
+name = astor
+description = Read/rewrite/write Python ASTs
+long_description = file:README.rst
+author = Patrick Maupin
+email = pmaupin@gmail.com
+platforms = Independent
+url = https://github.com/berkerpeksag/astor
+license = BSD-3-Clause
+keywords = ast, codegen, PEP8
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: Implementation
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Code Generators
+    Topic :: Software Development :: Compilers
+
+[options]
+zip_safe = True
+include_package_data = True
+packages = find:
+
+[options.packages.find]
+exclude = tests
+
 [wheel]
 universal = 1
+
+[build-system]
+requires = ["setuptools", "wheel"]
```

Comment:
Style nit: Use single quotes.

Reply:
This section is not part of setup.cfg spec, it's part of pyproject.toml one, but I expect them be merged into a single spec somewhen. TOML syntax requires double-quoted strings.

Reply:
OK